source/openapi3/split.yaml

```
@@ -2309,12 +2090,12 @@ paths:
 
         <aside class="notice">
           To enable this feature, please contact our support team with the following information:
-            <li>Your full legal account name</li>
+            <li>Your full legal business name</li>
             <li>A legally owned domain name: for your PayID email addresses</li>
-            <li><strong>alias_name</strong>: the short business name that will be displayed to your customers</li>
+            <li><strong>alias_name</strong>: a shortened business name that will be displayed to your customers upon PayID resolution</li>
```

What if the business name is already short?
```suggestion
            <li><strong>alias_name</strong>: a potentially shortened business name that will be displayed to your customers upon PayID resolution</li>
```